packages/local_auth/android/src/main/java/io/flutter/plugins/localauth/LocalAuthPlugin.java

```diff
@@ -67,136 +95,255 @@ public LocalAuthPlugin() {}
 
   @Override
   public void onMethodCall(MethodCall call, final Result result) {
-    if (call.method.equals("authenticateWithBiometrics")) {
-      if (authInProgress.get()) {
-        // Apps should not invoke another authentication request while one is in progress,
-        // so we classify this as an error condition. If we ever find a legitimate use case for
-        // this, we can try to cancel the ongoing auth and start a new one but for now, not worth
-        // the complexity.
-        result.error("auth_in_progress", "Authentication in progress", null);
-        return;
-      }
+    this.result = result;
+    switch (call.method) {
+      case "authenticate":
+        this.authenticate(call, result);
```

Comment:
`this.` qualifier is unnecessary (ditto below).

Reply:
Updated